functions/volume_level.h

```diff
@@ -0,0 +1,20 @@
+#ifndef FUNCTIONS_VOLUME_LEVEL_H
+#define FUNCTIONS_VOLUME_LEVEL_H
+
+#include "svf.h"
+
+// Usage: VolumeLevel
+// Returns 0-32768 based on volume level.
+// returned value: INTEGER
+
+class VolumeLevelSVF {
+ public:
+  void run(BladeBase* blade) {}
+  int calculate(BladeBase* blade) {
+    return clampi32((dynamic_mixer.get_volume() * 100 / VOLUME) * 32768 / 100, 0, 32768);
```

Comment:
no point in multiplying by 100 and dividing by it later.
Also, I don't think we need the clamp, I think this is enough:

   return dynamic_mixder.get_volume() * 32768 / VOLUME;